projects/canopy/src/lib/forms/sort-code/sort-code.component.ts

```diff
@@ -120,6 +113,19 @@ export class LgSortCodeComponent implements OnInit, ControlValueAccessor {
       this.ngControl.control.updateValueAndValidity();
     }
 
+    const sortCodeParentControl = this.parentFormGroupDirective.form.controls['sortCode'];
```

Comment:
I guess we need to import the name of "sortCode" to the users. Maybe something to add to the notes?

Reply:
Very good point. Added a new line in the sort-code.notes.